src/views/Hero/index.view.tsx

```diff
@@ -16,11 +16,17 @@ const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
     <div
       className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
     >
-      <PostcardStack pageName={pageName} />
-      <EmailSubscriptionForm />
-      Hello from HeroView
-      {title}
-      {description}
+      <div className="HeroView__container">
+        <div className="HeroView__visual">
+          <PostcardStack pageName={pageName} />
+        </div>
+        <div className="HeroView__text">
```

Comment:
Super nit-picky, but suggest changing the class name to `HeroView__textContainer` just so we can tell it apart from something else that might be called `text`. This is more of a container.